### DISCUSSION TITLE
Ikemen-GO won't launch on my Mac

### DISCUSSION THREAD

**Discussion by MugenMan10025** [ikemen-engine/Ikemen-GO#1651] [Q&A] [answered]
Every time I try to launch Ikemen-GO it says that I don't have the permission to access it. How do I get permission to open it?

**Comment by K4thos** [comment, marked as answer]
https://support.apple.com/en-gb/guide/terminal/apdd100908f-06b3-4e63-8a87-32e71241bab4/mac
